pkg/kafka/consumer/config.go

```diff
@@ -36,10 +37,11 @@ type Builder interface {
 
 // Configuration describes the configuration properties needed to create a Kafka consumer
 type Configuration struct {
-	Brokers  []string
-	Topic    string
-	GroupID  string
-	ClientID string
+	Brokers              []string
+	Topic                string
+	GroupID              string
+	ClientID             string
+	KafkaProtocolVersion string
```

Comment:
it's not necessary to quality this with Kafka prefix - the whole thing is Kafka configuration.